lua/kommentary/config.lua

```diff
@@ -71,6 +71,25 @@ function M.has_filetype(filetype)
     return M.config[filetype] ~= nil
 end
 
+function M.config_from_commentstring(commentstring)
+    if commenstring == "/*%s*/" then return default end
+
+    local placeholder = '%s'
+    local where = commentstring:find(util.escape_pattern(placeholder))
+
+    if not where then
+        return default
+    else
+        where = where - 1
+    end
```

Comment:
```suggestion
    if not where then
        return default
    end
    where = where - 1
```

The else block is unnecessary.